### DISCUSSION TITLE
Running into sizing issue with new image component

### DISCUSSION THREAD

**Discussion by JM-Price** [vercel/next.js#21230] [Help]
Newer dev here - I'm trying to use the image component and running into issues with the `layout` and `objectFit` properties. 

In this case, I'm wanting to have a logo in the nav bar surrounded by a border, but cannot seem to preserve the logo's original height/width properties with the <Image> component.  Here are is my code for the native <img> component that are working as desired along with an attached screenshot of the outcome.  When I use the <Image> component, it requires either height AND width, or layout=fill which warps the logo.  

I'm starting this discussion because I can see this becoming an issue for other cases.  I don't want to always rely on using height and width in pixels because that doesn't scale well or preserve aspect ratio. Sometimes %, REM, VW, VH, etc. are necessary for setting width and then height being set by aspect ratio. 

Any suggestions? Am I missing something? Is this a situation where I wouldn't want to use this image component and just use the default?

Code:
```
<Flex
    border={`3px solid ${colors.white50}`}
    height={'2.5rem'}
    padding={'3px 6px'}
    position={'relative'}
 >
    <img
        alt={'logo'}
        height={'100%'}
        src={'/assets/Logo.png'}
    />
</Flex>
```
<img width="252" alt="Screen Shot 2021-01-16 at 11 20 15 AM" src="https://user-images.githubusercontent.com/55364976/104820382-93014180-57f1-11eb-897c-0c3d41517a04.png">

**Comment by Sandstedt** [comment]
@JM-Price 

When the browser parse the HTML, it hasn't yet doing any layout calculations, so you can't use relative units, if I remember correctly. But anyway. The width and height of the <img>/<Image> component should we either the actual size of the logo, or a ratio that is identical as the logo. Ex:

```js
<Image
   src={'/assets/Logo.png'}
   width="400"
   height="150"
  >
```

now you can with CSS set either height to a fixed/relative height (ex 2.5rem) and width to auto, or width to a fixed width and height to auto. This will keep the original image ratios intact.

So nothing more special with this component than any other use of a default html <img> tag.
